### ISSUE TITLE
SQL error when moving channel without users

### ISSUE THREAD

**Issue by vpecinka** [mattermost/mattermost-server#9788]
#### Summary
When moving channel without members from one team to another I got an SQL error:

#### Steps to reproduce
Have and channel with no member and try it move e.g. like this
```
./bin/mattermost channel move vyhledavani odpovidac:admins5-alerts --username vlastimil.pecinka
```

#### Expected behavior
It should pass as it is valid having no-member channels (system admin still can add members later).

#### Observed behavior (that appears unintentional) 
I got this error:
```
Unable to move channel 'admins5-alerts' error: SqlTeamStore.GetMembersByIds: Unable to get the team members, teamId=twb8s5g1kpfk3ede1faihjihce Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ') AND TeamMembers.DeleteAt = 0' at line 11
```

**Comment by amyblais** [comment]
@hmhealey Would you be able to help test this out? (pinging you as the dev on support rotation for this week)

**Comment by hmhealey** [comment]
I'll take a look. I assume it's panicking because it's trying to make sure everyone in the channel is on the new team, but since there's no one in the channel, it breaks some logic around that

**Comment by hmhealey** [comment]
Yeah, I can reproduce it. I'll file a ticket in Jira to track it further.

A workaround for this in the mean time would be to add a user that's on both teams to the channel since that should fix the error
